packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts

```diff
@@ -71,8 +73,68 @@ export class HorizontalBarChart extends FASTElement {
     this.hydrateLegends();
   }
 
+  public enableTabbability() {
+    requestAnimationFrame(() => {
+      const legendContainer = this.shadowRoot?.getElementById('legend-div') as HTMLElement;
+      if (legendContainer) {
+        const legendButtons = tabbable.tabbable(legendContainer);
+        let currentLegendIndex = 0;
+
+        legendContainer.addEventListener('keydown', (event: KeyboardEvent) => {
+          // Only handle Arrow keys for legend buttons
+          if (legendButtons.length > 0) {
+            currentLegendIndex = this.handleArrowNavigation(event, legendButtons, currentLegendIndex, this._isRTL);
+          }
+        });
+      }
+      // Handle navigation for tabbable elements inside root div
+      const rootDiv = this.shadowRoot?.getElementById('root-div');
+      if (rootDiv) {
+        const tabbableElements = tabbable.tabbable(rootDiv);
+        let currentIndex = 0;
+
+        rootDiv.addEventListener('keydown', (event: KeyboardEvent) => {
```

Comment:
does pressing tab key still move focus to the next bar segment or does it move to elements in the next focusable group?

Reply:
It still moves to the next bar segment. Moving to the next bar is the functionality of Groupper which needs to be implemented separately like Arrow navigation.